### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by Marton6**

### CODE REVIEW

File: website/docs/r/secret.html.markdown [hunk 1]
```diff
@@ -8,7 +8,7 @@ description: |-
 
 # Resource: secrethub_secret
 
-This resource allows to write secrets at a given path, if the path is exists already then the resource will write a new version of it.
+This resource allows writing secrets at a given path, if the path already exists then the resource will write a new version of it.
```
Comment (SimonBarendse) [inline website/docs/r/secret.html.markdown]: I think `to write` is correct, isn't it?

Comment (Marton6) [reply website/docs/r/secret.html.markdown]: I think both are correct, but personally:
`This resource allows writing secrets...` or
`This resource allows you to write secrets...`
 sound better than
`This resource allows to write secrets...`

File: website/docs/r/service_aws.html.markdown [hunk 2]
```diff
@@ -21,3 +21,9 @@ The following arguments are supported:
 * `repo` - (Required) The path of the repository on which the service operates.
 * `role` - (Required) The role name or ARN of the IAM role that should have access to this service account.
 
+## See also
+
+- [AWS Integration](https://secrethub.io/docs/reference/aws/)
+- [AWS EC2 Guide](https://secrethub.io/docs/guides/aws-ec2/)
+- [AWS ECS Guide](https://secrethub.io/docs/guides/aws-ecs/)
+- [AWS Lambda (Golang) Guide](https://secrethub.io/docs/guides/aws-lambda-go/)
```
Comment (florisvdg) [inline website/docs/r/service_aws.html.markdown]: This one is not Terraformed yet, so I'd say leave it out for now.

File: website/docs/r/secret.html.markdown (hunk 1 above)
Comment (SimonBarendse) [reply website/docs/r/secret.html.markdown]: I like this one a lot: `This resource allows you to write secrets`.
By using you, it focuses on the user.

Comment (Marton6) [reply website/docs/r/secret.html.markdown]: Good point. I will use that phrasing.

**Pull request merged**